terraform/Agent.json.template

```diff
@@ -1,22 +1,24 @@
 {
   "name": "consul-agent",
   "image": "consul:${consul_version}",
+  "privileged": true,
```

Comment:
guessing this is to bind to port 53?

Reply:
yes

Reply:
i see the port 53 binding on the `Server` template, not not the `Agent`. is the `Server` the only one that actually needs this?

Reply:
It makes things a lot easier for the `Server` specifically because dealing w the Docker bridge IP is a pain. As for the Agent, technically containers local to it can use the Agent's exposed dns port for resolution as well. As such, I considered having the `Agent` follow the same port mapping pattern as `Server` but I opted not to in order to keep changes to a minimum. Could be convinced otherwise, though...

Reply:
well this does make sense if you are using the Agent specifically for DNS service discovery... so yeah... i'll buy that... but an example of that in your getting started would be cool